_shared/project/Makefile

```diff
@@ -219,34 +195,18 @@ docker-run:
 .PHONY: clean
 $(call help,make clean,"delete temporary files etc")
 clean:
-	@rm -rf build dist .tox
+	@rm -rf build dist .tox .coverage coverage .eslintcache node_modules supervisord.log supervisord.pid yarn-error.log
```

Comment:
I found a bunch more files that need to be deleted that were missing from `make clean`